source/python_library.rst

```diff
@@ -28,7 +28,7 @@ To add an additional element, create it, and group it with the VGroup.
 Text Box
 --------------------
 
-Our Text Box takes in:
+Text Box's takes in:
```

Comment:
Text Boxes take in?
Or did you mean the text box's take ins?

Reply:
Yeah I meant that but it's better without the comma actually